tests_new/integration_tests/conftest.py

```diff
@@ -14,6 +14,7 @@
 pytest_plugins = [
     'integration_tests.core.organizations.global_conftest',
     'integration_tests.core.environment.global_conftest',
+    'integration_tests.modules.share_base.global_conftest',
```

Comment:
global conftests are meant to be used by the other modules. I am not sure we will want all the configs that we are creating in the other modules. Maybe we can have 2 conftest, one `global_conftest` (exportable to other modules) and a plain `conftest` (used internally in the shares tests). This is important because the share1, share3 states are very unpredictable